test/npu-xrt/packet_flow/run.lit

```diff
@@ -0,0 +1,14 @@
+// (c) Copyright 2023 Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// REQUIRES: ryzen_ai
+//
+// RUN: %python aiecc.py %S/aie.mlir
+// RUN: aie-translate --aie-generate-cdo aie.mlir.prj/input_physical.mlir
+// RUN: cp *.elf aie.mlir.prj/
+// RUN: cp *.bin aie.mlir.prj/
```

Comment:
What's this?

Reply:
Legacy lit test example copied from outdated test files. Updated all of those with `aiecc.py --aie-generate-cdo`.